include/ocpp/v201/ocpp_types.hpp

```diff
@@ -861,6 +861,10 @@ void from_json(const json& j, NetworkConnectionProfile& k);
 /// \returns an output stream with the NetworkConnectionProfile written to
 std::ostream& operator<<(std::ostream& os, const NetworkConnectionProfile& k);
 
+struct ConfigNetworkResult {
```

Comment:
Don't forget to document this, that the interface address can be an ip address string or interface name ("192.168.42.1" or "eth6" for example).